From 3b9f847b4692d34f966af368a63d03887be11a79 Mon Sep 17 00:00:00 2001
From: Mamoru TASAKA <mtasaka@fedoraproject.org>
Date: Sun, 16 Nov 2014 16:55:48 +0900
Subject: [PATCH] binaryring.c: include stdint
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

On Fedora 21, binaryring.c fails to compile as:

../../hacks/binaryring.c:40:9: error: unknown type name ‘uint32_t’
 typedef uint32_t pixel_t;
         ^
Makefile:542: recipe for target 'binaryring.o' failed

Just include stdint.h for this type.
---
 hacks/binaryring.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hacks/binaryring.c b/hacks/binaryring.c
index 2432073..e3a58d9 100644
--- a/hacks/binaryring.c
+++ b/hacks/binaryring.c
@@ -27,6 +27,8 @@
 #include "colors.h"
 #include "hsv.h"
 
+#include <stdint.h>
+
 #define ANTIALIAS   1
 #define BLACK       0
 #define WHITE       1
-- 
2.1.0

